yarn.lock

```diff
@@ -1756,12 +1784,14 @@ cssstyle@^1.0.0:
 currently-unhandled@^0.4.1:
   version "0.4.1"
   resolved "https://registry.yarnpkg.com/currently-unhandled/-/currently-unhandled-0.4.1.tgz#988df33feab191ef799a61369dd76c17adf957ea"
+  integrity sha1-mI3zP+qxke95mmE2nddsF635V+o=
```

Comment:
Not sure where these are coming from. What version of yarn are you using?

There's a `.yvmrc` file to use with yvm.

Reply:
I'm using `yarn@1.13.0`